R/db-odbc-mssql.R

```diff
@@ -68,7 +68,28 @@
       as.double     = sql_cast("NUMERIC"),
       as.character  = sql_cast("VARCHAR(MAX)"),
       log           = sql_prefix("LOG"),
+      nchar         = sql_prefix("LEN"),
+      atan2         = sql_prefix("ATN2"),
+      substr        = sql_prefix("SUBSTRING"),
+      ceil          = sql_prefix("CEILING"),
       ceiling       = sql_prefix("CEILING"),
+      # Casting as BIT converts the Integer result into an actual logical
```

Comment:
Yes sir, here it is:

``` 
Select mpg is null as test from mtcars
```

```
Error in new_result(connection@ptr, statement) : 
  nanodbc/nanodbc.cpp:1344: 42000: Incorrect syntax near the keyword 'is'., Statement(s) could not be prepared.,  
Failed to execute SQL chunk
```
I also looked at `isnull`, but needs a replacement value, so it's not helpful

```
Select isnull(mpg, 1)  as test from mtcars
```